.github/actions/scan-links/action.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Scan Links'
+
+description: 'Scans markdown files for relative links extending outside the folder.'
+
+inputs:
+  include:
+    description: 'Directory to scan for markdown files. Relative to the repository root.'
+    required: false
+  base_directory:
+    description: 'Base directory which links must not reach out of. Relative to the repository root.'
+    required: false
+  ignore_paths:
+    description: 'Comma-separated list of paths to ignore during the scan. Relative to the repository root.'
+    required: false
+
+runs:
+  using: 'node20'
```

Comment:
Should we be using a later version of node? I thought 20 was being phased out for runners